glusterd2/commands/volumes/commands.go

```diff
@@ -134,6 +134,12 @@ func (c *Command) Routes() route.Routes {
 			Pattern:     "/volfiles",
 			Version:     1,
 			HandlerFunc: volfilesListHandler},
+		route.Route{
+			Name:        "VolfilesGet",
+			Method:      "GET",
+			Pattern:     "/volfiles/{volfileid:.*}",
```

Comment:
Some volfile ids includes "/" in it. Is that supported by this? For example, "gluster/glustershd"

Reply:
yes its supported